packages/react-log-viewer/src/LogViewer/examples/realTestData.js

```diff
@@ -1,6 +1,5 @@
 export const data = {
   data: `Copying system trust bundle
-
```

Comment:
I have added that empty line in #6033 as an easy way to spot regression :)

Reply:
okay. I moved it down a few lines so there isn't a blank space on the loading view. I don't want that to confuse people.